src/sdk-extras/Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG OSXCROSS_VERSION=12.3
+
+# Make sure to read XCode license agreement before using this code
+# same for all arch therefore hardcoded to arm64
+FROM --platform=darwin/arm64 crazymax/osxcross:${OSXCROSS_VERSION} AS osxcross
+
+FROM scratch AS sdk-extras-darwin
+COPY --from=osxcross /osxsdk /xx-sdk/
+
+# this dance is a hack to work aroung windows tarball being different from linux
+# create directories in both levels so layers can be accessed from either OS
+FROM --platform=$BUILDPLATFORM alpine AS extras-base
+RUN mkdir -p /out/Files/xx-sdk /out/xx-sdk
+
+FROM scratch AS sdk-extras-linux
+COPY --from=extras-base /out/Files /xx-sdk
+
+FROM scratch AS sdk-extras-windows
+COPY --from=extras-base /out /
+
+FROM sdk-extras-linux AS sdk-extras-freebsd
+
+FROM sdk-extras-${TARGETOS} AS sdk-extras
```

Comment:
Not sure why we need this Dockerfile, you can directly use `crazymax/osxcross` image as it already carries dummy platforms with respected folders so it doesn't fail on cross for unsupported platforms: https://github.com/crazy-max/docker-osxcross/blob/4b6234e77bdae7375da23b728ebba73a49453e99/Dockerfile#L14-L25

See https://hub.docker.com/r/crazymax/osxcross/tags

![image](https://user-images.githubusercontent.com/1951866/218564436-f866f336-c220-48aa-bea8-12e07c55e5e0.png)

Reply:
the path is wrong in that image. `xx` expects `xx-sdk` and I think that one had osxcross as well that `xx` does not need.